### DISCUSSION TITLE
Issues with converting to free form.

### DISCUSSION THREAD

**Discussion by dhfreinkel** [codefori/vscode-ibmi#957] [IBM i] [answered]
Converting Procedure Interfaces (PI), it is not converting correctly.

        Dcl-PI jrn_fix_02  extpgm('JRN_FIX_02');
          Lib_to_Check   Char(10a);
          New_Jrn        Char(10a);
          New_Jrn_Lib    Char(10a);
        End-PI;
It correctly changes to CHAR() but the a should be removed.

Old Code: 
     d wfield                 10a

The other issue I found is it randomly adds End-PR:

**Comment by worksofliam** [comment, marked as answer]
Hi @dhfreinkel 

Your issue should be raised here: https://github.com/BrianGarland/vscode-rpgfree

This extension is not maintained by our core team.
